src/integration.test.ts

```diff
@@ -3,7 +3,7 @@ import { MAX_GET_ENTRY_TIMEOUT } from "./registry";
 import { MAX_REVISION } from "./utils";
 
 // TODO: Use siasky.dev when available.
```

Comment:
This line can go now. ;)

Reply:
Looks like siasky.dev is still not working (the registry is broken), going back to siasky.net for now